libs/base/Data/Stream.idr

```diff
@@ -97,15 +97,15 @@ takeUntil : (n -> Bool) -> Stream n -> List n
 takeUntil p (x :: xs)
     = if p x
          then [x]
-         else x :: takeUntil p xs
+         else x :: Stream.takeUntil p xs
 
 public export
 partial
 takeBefore : (n -> Bool) -> Stream n -> List n
 takeBefore p (x :: xs)
     = if p x
          then []
-         else x :: takeBefore p xs
+         else x :: Stream.takeBefore p xs
```

Comment:
Data.Stream `takeUntil` and `takeBefore` should probably be deduplicated since it was defined in Prelude and needs to be `public export` to get some `auto` searches of `Range`s to work.

Reply:
Why don't you remove them? `Prelude` is automatically loaded so these will always be clashing.

Reply:
Amended